crates/humanode-rpc/src/lib.rs

```diff
@@ -86,7 +87,7 @@ pub struct EvmDeps {
     /// Maximum number of stored filters.
     pub eth_max_stored_filters: usize,
     /// Backend.
-    pub eth_backend: Arc<fc_db::Backend<Block>>,
+    pub eth_backend: Arc<dyn fc_db::BackendReader<Block> + Send + Sync>,
```

Comment:
Why `dyn` trait here instead of a generic parameter?

Reply:
As `eth_backend` should implement `fc_db::BackendReader<Block>` trait.

Reply:
Well, this is true, but still - why dyn trait? Is a dyn trait required somewhere down the line?

Reply:
https://github.com/humanode-network/humanode/pull/869

Reply:
Checked why `dyn`. It's explicitly required here - https://github.com/humanode-network/frontier/blob/9f01d682957d0a2bcc022e0f0264941342fb5305/client/rpc/src/eth/mod.rs#L83.